### ISSUE TITLE
Asia

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#191]
Asia Accessibility Remediation

Number of issues: 42
Current score: 54.6%
PDF errors: 100

**Comment by kevinkidwell** [comment]
@manolobevia @mhk33 I've been going through the accessibility for Asia, in particular the contrast issues which were the biggest contributor to design change. I've pushed the changes to accommodate this to https://dev-asialibrarycornelledu.pantheonsite.io/

The biggest change is right below the header with removing that yellow color background. It could have stayed, but it would have needed to be such a deep brown with white text it really didn't look right to me. I still have some hover colors to fix but wanted to get this in front of both of you to see if this works or if there are any other little things that you should be adjusted with it.

**Comment by manolobevia** [comment]
Once you removed the background color for the title of the page the title is gone.
![asia](https://user-images.githubusercontent.com/8395482/55827062-56aa9880-5ad7-11e9-9a84-fd63a7b3d1de.png)

**Comment by mhk33** [comment]
I like what you've done, @kevinkidwell -- I didn't really notice the difference at first.

**Comment by manolobevia** [comment]
@kevinkidwell I was going to update modules and core on this drupal site and I see the following issues:

**Comment by manolobevia** [comment]
In mobile devices, the nav doesn't have any content. 

![Screen Shot 2019-08-06 at 10 21 00 AM](https://user-images.githubusercontent.com/8395482/62548229-7909b400-b834-11e9-954c-d878d896c622.png)

**Comment by manolobevia** [comment]
There is duplication of headers all over the site.
![Screen Shot 2019-08-06 at 10 21 29 AM](https://user-images.githubusercontent.com/8395482/62548309-976faf80-b834-11e9-8837-c7917eebd5ca.png)
